commit f11268b774d8b5bc4c8cd224751669bd9be67626 (refs/changes/28/1428/2)
Author: David Pacheco <dap@joyent.com>
Date:   2017-02-03T15:02:04-08:00 (2 years, 8 months ago)
    
    MANTA-3137 minnow alarms fire spuriously after MANTA-3127
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/probes/storage/gossip.json b/probes/storage/gossip.json
index 519fdb3..3f8f52c 100644
--- a/probes/storage/gossip.json
+++ b/probes/storage/gossip.json
@@ -3,7 +3,7 @@
         "type": "cmd",
         "name": "minnow heartbeat too old",
         "config": {
-            "cmd": "/bin/bash -c 'let delta=$(date +%s)-$(PATH=/opt/smartdc/minnow/build/node/bin:/opt/smartdc/minnow/node_modules/.bin:$PATH findobjects -h $(cat /opt/smartdc/minnow/etc/config.json | json moray.host) manta_storage hostname=$(hostname)* | json -e _mtime=_mtime/1000 -e _mtime=~~_mtime _mtime) ; test $delta -lt 900'",
+            "cmd": "/bin/bash -c 'if [[ -f /opt/smartdc/minnow/bin/check-minnow ]]; then /opt/smartdc/minnow/bin/check-minnow; exit $?; fi; let delta=$(date +%s)-$(PATH=/opt/smartdc/minnow/build/node/bin:/opt/smartdc/minnow/node_modules/.bin:$PATH findobjects -h $(cat /opt/smartdc/minnow/etc/config.json | json moray.host) manta_storage hostname=$(hostname)* | json -e _mtime=_mtime/1000 -e _mtime=~~_mtime _mtime) ; test $delta -lt 900'",
             "threshold": 3,
             "period": 300,
             "timeout": 30
